.github/workflows/acceptance_tests.yml

```diff
@@ -0,0 +1,139 @@
+name: Acceptance tests
+on:
+  push:
+    branches:
+      - develop
+      - release-*
+  pull_request:
+    branches:
+      - develop
+      - release-*
+
+jobs:
+  build:
```

Comment:
Why don't we use the build generated by e2e tests?

Reply:
Yes, that is a great idea. It will save us from unnecessary extra builds. But we need to make sure that e2e tests run before the acceptance tests. Doesn't all workflow run asynchronously at the same time ?

Reply:
@seanlip done

Reply:
I agree with @vojtechjelinek on this. It will be a good idea to have a single build. @ashish-patwal can you move this to the e2e test workflow?

The flow is:
![image](https://user-images.githubusercontent.com/30050862/230568897-7d614606-03c8-4589-84de-4cc4a226e6a8.png)

So the build takes place and then all e2e test suites and lighthouse tests run asynchronously.

Reply:
Created an issue to address this in a new PR. https://github.com/oppia/oppia/issues/17975